frontend/src/components/dashboard/Dashboard.js

```diff
@@ -253,14 +254,15 @@ export class Dashboard extends Component {
   }
 
   render() {
+    console.log(this.props.mineRegionOptions);
```

Comment:
Remove console.log

Reply:
went through and removed them all this am.